policies/api/http/logging.go

```diff
@@ -284,6 +284,20 @@ func (l loggingMiddleware) DeleteSinkFromAllDatasetsInternal(ctx context.Context
 	return l.svc.DeleteSinkFromAllDatasetsInternal(ctx, sinkID, ownerID)
 }
 
+func (l loggingMiddleware) DeleteAgentGroupFromAllDatasets(ctx context.Context, aGroupID string, token string) (err error) {
+	defer func(begin time.Time) {
+		if err != nil {
+			l.logger.Warn("method call: delete_agroup_from_all_datasets",
```

Comment:
I like to keep things consistent, and I don't recall us using "agroup" to abbreviate "agent group" before. What do we do in other places? Just "group" or "agent_group"?